target/linux/brcm63xx/image/Makefile

```diff
@@ -132,11 +132,6 @@ define Build/spw303v-bin
 	mv $@.spw303v $@
 endef
 
-define Build/xor-image
```

Comment:
@musashino205 This is good idea in itself, but hidden in the patch for new board in `ath79` platform, althought touching code in `brcm63xx` platform. So it probably belongs to separate patch for reusable `xor-image` image target.